### PR TITLE
[Jenkins 28843] fix scm and review comments.

### PR DESCRIPTION
**Pull request by jtnord**
Update the SCM information for release.
Address a few comments from @jglick that came a little late for #1

@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
   <scm>
     <connection>scm:git:ssh://github.com/jenkinsci/unique-id-plugin.git</connection>
-    <developerConnection>scm:git:ssh://git@github.com/jenkinsci/unique-id-plugin.git</developerConnection>
+    <developerConnection>scm:git:https://github.com/jenkinsci/unique-id-plugin.git</developerConnection>
```
Comment (jglick) [inline pom.xml]: AFAIK the canonical block here is

```xml
    <scm>
        <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
        <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
        <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
    </scm>
```

File: src/main/java/org/jenkinsci/plugins/uniqueid/impl/IdStoreMigratorV1ToV2.java [hunk 2]
```diff
@@ -9,6 +9,7 @@
 import hudson.model.Run;
 
 import jenkins.model.Jenkins;
+import jenkins.util.io.FileBoolean;
```
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/uniqueid/impl/IdStoreMigratorV1ToV2.java]: I suppose this is a reminder for the future?

**Comment by jglick** [comment]
:+1:

File: pom.xml (hunk 1 above)
Comment (jtnord) [inline pom.xml]: I was under the impression that
`git@github.com` used ssh keys for auth.
when releasing on the CI we only have tokens that are not keys and can only be used with user/token auth over https?

File: src/main/java/org/jenkinsci/plugins/uniqueid/impl/IdStoreMigratorV1ToV2.java (hunk 2 above)
Comment (jtnord) [inline src/main/java/org/jenkinsci/plugins/uniqueid/impl/IdStoreMigratorV1ToV2.java]: no :(

**Comment by tfennelly** [comment]
:+1:

File: pom.xml (hunk 1 above)
Comment (jglick) [inline pom.xml]: > when releasing on the CI

Oh well I have never tried to release plugins from jenkins.ci.cloudbees.com if that is what you mean. Making that really work properly is an open topic (@KostyaSha and I were discussing possibilities).

Comment (jtnord) [inline pom.xml]: - define properly - m2release works :-)

**Pull request merged**